src/components/Dasboard/Cards/DasboardCards.tsx

```diff
@@ -6,9 +6,11 @@ import { FormattedMessage } from "react-intl";
 export default function DashboardCard({
 	title = "",
 	children,
+	cardHeight = "11rem",
```

Comment:
Instead of asking just the card height can we ask for a style object for card container?